includes/sdk/FreemiusWordPress.php

```diff
@@ -333,6 +342,30 @@ private static function ExecuteRequest( $pUrl, &$pWPRemoteArgs ) {
 			return $response;
 		}
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         *
+         * @param string $pUrl
+         * @param array  $pWPRemoteArgs
+         *
+         * @return mixed
+         */
+        static function RemoteRequest( $pUrl, $pWPRemoteArgs ) {
+            $response = wp_remote_request( $pUrl, $pWPRemoteArgs );
+
+            if (
+                ! is_wp_error( $response ) &&
+                (
+                    empty( $response['headers'] ) ||
+                    empty( $response['headers']['x-api-server'] )
+                )
+            ) {
+                $response = new WP_Error( 'api_blocked', htmlentities( $response['body'] ) );

```

Comment:
Please add a comment explaining the headers check and why we consider it as a blocked API.